extensions/agroal/runtime/src/main/resources/META-INF/quarkus-extension.json

```diff
@@ -0,0 +1,8 @@
+
+{
+    "name": "Agroal - Database connection pool",
+    "labels": [
```

Comment:
I'm wondering if we should include at least part of the info we have on the `extensions.yaml` file from the website?

See https://github.com/quarkusio/quarkusio.github.io/blob/develop/_data/extensions.yaml .

I'm thinking about:
- the description
- a key defining the category (I would keep the list of category key <-> name mapping and the order in the website - or on `code.quarkus.io`).

If we do that, I wouldn't have to add the extensions on the website.

Note that I haven't followed all the discussions so maybe it's a stupid idea :).

Reply:
Sure. Here I'm taking what's in the current `extensions.json` as a start.

Reply:
+1 on getting the minimal version in first.
Next step there efter is to add the additional features such as description, category, etc.